core/src/test/java/org/pentaho/platform/util/StringUtilTest.java

```diff
@@ -75,14 +92,20 @@ public void testDoesPathContainParentPathSegment() {
     }
   }
 
-  public static void main( final String[] args ) {
-    StringUtilTest test = new StringUtilTest();
+  @Test
+  public void testMain() {
     try {
-      test.testTokenStringToArray();
-      test.testDoesPathContainParentPathSegment();
-
-    } finally {
+      Assert.assertTrue( true );
```

Comment:
Can't the testMain() method simply be removed? All it does is test that asserting true works - that's not testing anything in StringUtil at all.